runtime-light/component/component.cpp

```diff
@@ -6,18 +6,104 @@
 
 #include <chrono>
 #include <cstdint>
+#include <memory>
 #include <utility>
 
 #include "runtime-core/utils/kphp-assert-core.h"
 #include "runtime-light/core/globals/php-init-scripts.h"
+#include "runtime-light/coroutine/awaitable.h"
+#include "runtime-light/coroutine/task.h"
 #include "runtime-light/header.h"
 #include "runtime-light/scheduler/scheduler.h"
+#include "runtime-light/streams/streams.h"
 #include "runtime-light/utils/context.h"
+#include "runtime-light/utils/json-functions.h"
+
+namespace {
+
+constexpr uint32_t K2_INVOKE_HTTP_MAGIC = 0xd909efe8;
```

Comment:
Maybe create one more file, that will contain init functions?

Reply:
I'll do it in the next PR.